definitions/bulk-action/json/ComplexTypes.json

```diff
@@ -49,11 +49,51 @@
     "SecurityClassification": "Public"
   },
   {
-    "LiveFrom": "01/01/2019",
+    "LiveFrom": "01/01/2017",
     "ID": "BulkListDocument",
-    "ListElementCode": "url",
+    "ListElementCode": "DocumentType",
+    "FieldType": "Text",
+    "ElementLabel": "Type",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
```

Comment:
what is this for?

Reply:
@qzhou-hmcts, this is a replication of the complex type we have in the Divorce case type for a document. I did it like this so that we can reuse the same endpoint in COS.
This will hold the information about the generated PDF.